src/storage/batch_extractor.cc

```diff
@@ -232,6 +232,10 @@ rocksdb::Status WriteBatchExtractor::PutCF(uint32_t column_family_id, const Slic
               first_seen_ = false;
             }
             break;
+          case kRedisCmdBitfield:
```

Comment:
Would you mind testing the slot-migration case here?

Reply:
Could you tell me how to run only one test instead of all go test? I wrote the tests and I tried `./x.py test go build redis-cli -run <test function name>`, but no effect. Thanks.